### ISSUE TITLE
a11y > empty + / - button

### ISSUE THREAD

**Issue by ryanschaefer41** [vigetlabs/EcoCart#80]
Add title prop for +/- buttons.

If that doesn't work, [here's](https://gomakethings.com/hidden-content-for-better-a11y/#hiding-the-link) a potential alternate approach.
